src/components/collection_vote_box/CollectionSubmitVoteBox.tsx

```diff
@@ -181,14 +183,14 @@ const CollectionSubmitVoteBox: React.FC<CollectionSubmitVoteBoxProps> = (props)
     if (isChair && updateFinalVote && bucketKey) {
       updateFinalVote(bucketKey, { vote: newVote, rationale }, voteIds)
     }
-    else {
-      setVote(newVote)
-    }
+    setVote(newVote)
```

Comment:
This is a small bug fix. After updating a vote value on the back end, we should always set the local state so the button shows up correctly.